packages/salesforcedx-vscode-soql/README.md

```diff
@@ -1 +1,25 @@
-# SOQL Builder!
+# SOQL Builder for Visual Studio Code
+
+SOQL Builder enables you to interactively build a SOQL query via a form-based visual editor. View the query as you build, and save the output to a .csv or .json file.
+
+![GIF showing SOQL query builder, viewing query results, and saving results to csv](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-soql/images/SOQL_Builder.gif)
```

Comment:
This URL does not exist yet, but I think it will once this PR is pushed to develop.